cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_utils.go

```diff
@@ -143,3 +152,38 @@ func machineSetIsOwnedByMachineDeployment(machineSet *MachineSet, machineDeploym
 	}
 	return false
 }
+
+func scaleFromZeroEnabled(annotations map[string]string) bool {
+	if val, exists := annotations[nodeGroupScaleFromZero]; exists {
```

Comment:
can we make this implicit from the existence or not of the expected annotations? i.e 
https://github.com/openshift/cluster-api-provider-aws/pull/301/commits/5b559fa44240eb45743c17ac7116c206be0f38d8#diff-0a4645af87c5c248d6ad15211e1d3340R25-R27